task_12/app/js/script.js

```diff
@@ -0,0 +1,17 @@
+window.onload = startTime();
+
+function startTime() {
+    var today = new Date();
+    var h = today.getHours();
+    var m = today.getMinutes();
+    var s = today.getSeconds();
+    m = checkTime(m);
+    s = checkTime(s);
+    document.getElementById('current-time').innerHTML =
+    h + ":" + m + ":" + s;
+    var t = setTimeout(startTime, 500);
```

Comment:
It's better to use `setInterval` here because you do not have condition for recursion step/stop.